sources/packages/web/src/components/generic/formio.vue

```diff
@@ -76,6 +79,7 @@ export default {
         formDefinition.data,
         {
           fileService: new FormUploadService(),
+          readOnly: props.readOnly,
```

Comment:
did you check, what happens when `readOnly is ""` (when `readOnly` is not passed)?

Reply:
I would also be looking forward to knowing what would be the default value.
Whatever it is it could be the default value of the property also.

Reply:
Yes @ann-aot I tested, you can check it by running locally, the edit application, where the readonly is null as it is not passed